innkeeper/eskiptojson_test.go

```diff
@@ -0,0 +1,188 @@
+package innkeeper
+
+import (
+	"github.com/coreos/etcd/Godeps/_workspace/src/github.com/stretchr/testify/assert"
```

Comment:
please don't import packages from the Godeps workspace, neither ours, especially not from the imported packages' Godeps. IDE hiccup?

On the other hand: there was a reason why we didn't include testing frameworks. Please see, https://golang.org/doc/faq#testing_framework. In this particular case, I recommend to use table testing, that I already saw applied in other tests in this pull request.